project/website.md

```diff
@@ -0,0 +1,35 @@
+# Pacifica Main Website
+
+This is a space to develop some of the text required for the website.
+
+## Home Page
+
+This section is for the home page text, maybe links to images.
+
+### Top Page
+
+Collaborate. Share. Integrate. Build. 
+Pacifica supports the scientific data lifecycle.
+
+### Data Management
+
+Scientific Data Management systems for research and development
+institutions require many solutions to already complex problems.
+Pacifica has solutions to manage data between projects that are
+completing and collaborative. Pacifica can accelerate the scientific
+process by making search and discovery tracta
+
+### How to Solve it
+
+Open Source does not stop at releasing the source code. Many problems
+require community to share ideas, processes and support features to
+solve the requirements. This is the natural evolution of Open Source
+to Open Everything.
+
+### Impact
+
+Pacifica seeks to enable institutions to reduce risk while
+accelerating scientific impact of the data they produce. Pacifica has
```

Comment:
is it scientific impact for everyone who might use Pacifica?

Reply:
Done.